Packaging.props

```diff
@@ -19,6 +19,7 @@
     <!-- temporary change until buildtools can be updated https://github.com/dotnet/buildtools/issues/831 -->
     <LineupPackageVersion Condition="'$(LineupPackageVersion)' == ''">1.0.3</LineupPackageVersion>
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.0.2</PlatformPackageVersion>
+    <SkipValidatePackageTargetFramework>true</SkipValidatePackageTargetFramework>
```

Comment:
This was a change recommended by @ericstj.  This disables some validation that Eric is removing from the system anyway (it was logic to keep package versions internally consistent before we had fully rolled out generations) and the validation had some issues with not allowing dependencies to roll forward.

@ericstj is working on a larger change to remove all this infrastructure, but it was blocking us so we just disabled it for now.